lib/shelley/src/Cardano/Wallet/Shelley/Network.hs

```diff
@@ -1024,4 +1037,4 @@ instance HasSeverityAnnotation (NetworkLayerLog b) where
         MsgWatcherUpdate{}                 -> Debug
         MsgChainSyncCmd cmd                -> getSeverityAnnotation cmd
         MsgInterpreter{}                   -> Debug
-        MsgInterpreterPastHorizon{}        -> Critical
+        MsgInterpreterPastHorizon{}        -> Info
```

Comment:
Is this change really sound :thinking: ?

Reply:
I think the reasoning is that we have actually some cases where the PastHorizonException is "normal" and somewhat expected.

But perhaps we can divide slot/time conversions into two categories:
1. Where it's our bug if the conversion fails.
2. Where any conversion errors are because of the user inputting a date which is beyond the safe zone. So an error not a bug.